subsys/bluetooth/controller/ll_sw/ull_llcp_conn_upd.c

```diff
@@ -566,7 +672,19 @@ static void rp_cu_st_wait_rx_conn_param_req(struct ll_conn *conn, struct proc_ct
 {
 	switch (evt) {
 	case RP_CU_EVT_CONN_PARAM_REQ:
-		rp_cu_send_conn_param_req_ntf(conn, ctx, evt, param);
+		pdu_decode_conn_param_req(ctx, param);
+
+		bool parameters_is_changed = ull_conn_is_params_changed(conn,
```

Comment:
Nit-pick (capital N) spelling: parameters_ARE_changed or better jet just loose the 'is' (also in function!)

Reply:
Sure! Function and parameters renamed!